mkdocs.yml

```diff
@@ -17,6 +17,5 @@ pages:
             - "File Transport Options": transport/file-options.md
         - "Reading and Storing Mail": read.md
 site_name: zend-mail
-site_description: zend-mail
+site_description: 'Parse, create, store, and send email messages, using a variety of storage and transport protocols.'
```

Comment:
@froschdesign are we fine with using single quotes here? I've seen usually we go with double

Reply:
Both are working: https://pyyaml.org/wiki/PyYAMLDocumentation#scalars

The single quotes are correct here because no escaping is needed. But we were never very strict in this topic.